.gitignore

```diff
@@ -68,6 +68,7 @@ tests/framework/ROM/TimeSeries/ARMA/InterpolatedMaxCycles/
 tests/framework/ROM/TimeSeries/SyntheticHistory/LogARMA/
 tests/framework/ROM/TimeSeries/SyntheticHistory/VARMA/
 tests/framework/ROM/TimeSeries/SyntheticHistory/ZeroFilterDiscontinuous/
+*.xml.bak
```

Comment:
Make sure you don't commit any *.xml.bak files, but we shouldn't need to have this in the .gitignore file.